src/server/index.spec.tsx

```diff
@@ -697,6 +697,32 @@ describe(__filename, () => {
         expect(response.status).toEqual(200);
         expect(response.header).toHaveProperty('content-security-policy');
 
+        const policy = cspParser(response.header['content-security-policy']);
+        expect(policy['script-src']).toEqual([
+          "'self'",
+          "'unsafe-inline'",
+          `${devEnv.REACT_APP_ANALYTICS_HOST}${ANALYTICS_PATH}`,
+        ]);
+      });
+
+      it('uses debug analytics path when REACT_APP_GA_DEBUG_MODE is true', async () => {
+        const content = '<h1>It works!</h1>';
+        fakeCreateReactAppServerApp.get('/*', (req, res) => res.send(content));
+
+        const server = request(
+          createServer({
+            env: {
+              ...devEnv,
+              REACT_APP_GA_DEBUG_MODE: 'true',
```

Comment:
Are there already existing tests that check the opposite case, when it's 'false?

Reply:
No, there's isn't one. I'll add one. Thanks!

Reply:
[`3e80765` (#1641)](https://github.com/mozilla/addons-code-manager/pull/1641/commits/3e80765850bf2157c0f46df76a9404d1cb86ffeb)